models/comments.model.js

```diff
@@ -0,0 +1,33 @@
+const db = require('../db/connection');
+
+exports.fetchComments = (articleId) => {
+    return db.query(`
+        SELECT * 
+        FROM comments 
+        WHERE article_id = $1
+        ORDER BY created_at DESC;
+    `, [articleId])
+    .then((data) => {
+        return data.rows;
+    })
+    .catch((err) => {
+        return err;
+    })
```

Comment:
When using promises, we should only have one catch block at the end of the promise chain (in the controller). Please remove this localised catch block as it is unnecessary